config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get "/user-new", to: "users#new"
```

Comment:
bad name
em đổi lại như này nhé:
```suggestion
  get "/login", to: "users#new"
```